pennylane_lightning/lightning_kokkos/_state_vector.py

```diff
@@ -281,10 +284,26 @@ def _apply_lightning(
             elif method is not None:  # apply specialized gate
                 param = operation.parameters
                 method(wires, invert_param, param)
-            elif isinstance(operation, qml.ops.Controlled) and isinstance(
-                operation.base, qml.GlobalPhase
+            elif isinstance(operation, qml.ops.Controlled) and (
+                isinstance(
+                    operation.base,
+                    (
+                        qml.GlobalPhase,
+                        qml.PauliX,
+                        qml.PauliY,
+                        qml.PauliZ,
+                        qml.Hadamard,
+                        qml.S,
+                        qml.T,
+                        qml.PhaseShift,
+                        qml.RX,
+                        qml.RY,
+                        qml.RZ,
+                        qml.Rot,
```

Comment:
Do you think will be better to define this list at the top of the file or in `lightning_kokkos.py`?

Reply:
This list is temporary, and will be removed when a later PR when all gates are supported